wca-regulations.md

```diff
@@ -515,6 +519,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - F2) The judge places the scrambled puzzle onto the mat in a standing position.
     - F2a) The organization team may enforce using a stand for the puzzle to prevent it from falling before the start of the attempt. If such stands are used, the organization team should make an announcement before the round starts.
         - F2a1) The judge must remove the stand from the mat immediately after the competitor removes the puzzle from it.
+    - F2b) If, during the transportation phase or while placing the puzzle cover on the table, one or more of the pins have their state affected, the competitor may proceed with the attempt and, after finishing the solve, alongside with the judge, explain the situation to the WCA Delegate (see [Guideline F1a1+++](guidelines:guideline:F1a1+++)). If the WCA Delegate believes that the competitor did not gain an unfair advantage, they may allow the result to stand under their discretion.
```

Comment:
Nit: should this also include lifting the cover?
```suggestion
    - F2b) If the state of one or more of the pins is affected during the transportation phase or while placing the puzzle cover on the table, the competitor may proceed with the attempt and, after finishing the solve, alongside with the judge, explain the situation to the WCA Delegate (see [Guideline F1a1+++](guidelines:guideline:F1a1+++)). If the WCA Delegate believes that the competitor did not gain an unfair advantage, they may allow the result to stand under their discretion.
```

Reply:
I don't actually think F2b is necessary because of #1070 
```suggestion
```